code/API_definitions/number_verification.yaml

```diff
@@ -96,7 +96,7 @@ paths:
       description: |-
         Returns the phone number so the API clients can verify the number themselves:
          - It will be done for the user that has authenticated via mobile network and so their `sub` is in the access token
-         - It returns the authenticated user's `MSISDN` retrieved from the access token
+         - It returns the authenticated user's `device phone number` retrieved from the access token
```

Comment:
`device phone number` retrieved from the access token --> `device phone number` *associated to* the access token

Reply:
Edited